src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const carRentalCost = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
+
+  if (days >= 3 && days < 7) {
+    return days * carRentalCost - minimumDiscount;
+  }
+
+  if (days >= 7) {
+    return days * carRentalCost - maximumDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return days * carRentalCost - maximumDiscount;
  } else if (days >= 3) {
    return days * carRentalCost - minimumDiscount;
  }
```
If you swap these statements vice versa you can get rid of one redundant check.
Also, use `if - else if` construction